### PR TITLE
New package: MimecastLtd.MimecastforOutlook version 7.10.1.133

### PR DESCRIPTION
**Pull request by CalenCyr**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/135919)

### CODE REVIEW

File: manifests/m/MimecastLtd/MimecastforOutlook/7.10.1.133/MimecastLtd.MimecastforOutlook.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+PackageIdentifier: MimecastLtd.MimecastforOutlook
+PackageVersion: 7.10.1.133
+PackageLocale: en-US
+Publisher: Mimecast Ltd
+PublisherUrl: https://www.mimecast.com/
+PublisherSupportUrl: https://community.mimecast.com/s/contactsupport
+PackageName: Mimecast for Outlook
+PackageUrl: https://community.mimecast.com/s/mimecast-for-outlook
+License: Copyright (c) Mimecast Ltd. All rights reserved
```
Comment (Trenly) [inline manifests/m/MimecastLtd/MimecastforOutlook/7.10.1.133/MimecastLtd.MimecastforOutlook.locale.en-US.yaml:12]: A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate

File: manifests/m/MimecastLtd/MimecastforOutlook/7.10.1.133/MimecastLtd.MimecastforOutlook.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: MimecastLtd.MimecastforOutlook
+PackageVersion: 7.10.1.133
+InstallerLocale: en-US
+InstallerType: zip
+NestedInstallerType: wix
+Dependencies:
+  ExternalDependencies:
```
Comment (Trenly) [inline manifests/m/MimecastLtd/MimecastforOutlook/7.10.1.133/MimecastLtd.MimecastforOutlook.installer.yaml:10]: This should be a package dependency; External dependencies are not checked

**Review by Trenly — CHANGES_REQUESTED** [review]
When no outlook is installed, this is the error message on x64 systems - 
image

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 

(Automated response - build 646.)

**Pull request closed without merging**